_config.yaml

```diff
@@ -11,6 +11,16 @@ end_date: YYYY-MM-DD
 # Specific release in AlexsLemonade/training-modules
 # We use master to make development easier
 release_tag: master
+
+#### Workshop information ####
+workshop_type: WORKSHOP    # Update this value to either: "remote" or "in-person"
+workshop_content: WORKSHOP # Update this value to either: "intro-r", "intro-bulk", "intro-single-cell", or "advanced-single-cell"
+
+#### Instructor names ####
+# This variable should be filled with a list of team members leading a given workshop
+instructors: "`Ally Hawkins`, `Jen O'Malley`, `Josh Shapiro`, or `Stephanie Spielman`"
```

Comment:
I had a thought here to make this an actual array, but it turns out that joining a list with a terminal "or" is a bit ugly. We _could_ do something like this in the text if we wanted to though:

```
{% for name in site.instructors %}
{% if not forloop.last %}`{{name}}`, {%else%}or `{{name}}`{% endif %}
{% endfor %} 
```

The one think I don't like about the current method is that the formatting is baked into the string, which may not be what we want if we were going to reuse this elsewhere.